src/main/java/in/mohamedhalith/dao/EmployeeDAO.java

```diff
@@ -120,34 +121,42 @@ public Employee getEmployee(String username) throws DBException, ValidationExcep
 	 * @throws DBException
 	 * @throws ValidationException
 	 */
-	public boolean updateLeaveBalance(Employee employee, String type, int duration) throws DBException {
+	public boolean updateLeaveBalance(String keyword, int employeeId, LeaveRequest leaveRequest) throws DBException {
 		Connection connection = null;
 		PreparedStatement statement = null;
-		String username = employee.getUsername();
+		String type = leaveRequest.getType().toLowerCase();
```

Comment:
change the variable name type to leaveType or some other

Reply:
Ok